examples/SetBucketReplication.java

```diff
@@ -0,0 +1,111 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.minio.MinioClient;
+import io.minio.SetBucketReplicationArgs;
+import io.minio.errors.MinioException;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class SetBucketReplication {
+  /** MinioClient.setBucketReplication() example. */
+  public static void main(String[] args)
+      throws IOException, NoSuchAlgorithmException, InvalidKeyException {
+    try {
+      /* play.min.io for test and development. */
+      MinioClient minioClient =
+          MinioClient.builder()
+              .endpoint("https://play.min.io")
+              .credentials("Q3AM3UQ867SPQQA43P2F", "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG")
+              .build();
+
+      /* Amazon S3: */
+      // MinioClient minioClient =
+      //     MinioClient.builder()
+      //         .endpoint("https://s3.amazonaws.com")
+      //         .credentials("YOUR-ACCESSKEY", "YOUR-SECRETACCESSKEY")
+      //         .build();
+
+      /* Amazon S3 specific configuration. */
+      // String config =
+      //     "<ReplicationConfiguration>"
+      //         + "  <Role>arn:aws:iam::35667example:role/CrossRegionReplicationRoleForS3</Role>"
+      //         + "  <Rule>"
+      //         + "    <ID>rule1</ID>"
+      //         + "    <Status>Enabled</Status>"
+      //         + "    <Priority>1</Priority>"
+      //         + "    <DeleteMarkerReplication>"
+      //         + "       <Status>Disabled</Status>"
+      //         + "    </DeleteMarkerReplication>"
+      //         + "    <Filter>"
+      //         + "       <And>"
+      //         + "           <Prefix>TaxDocs</Prefix>"
+      //         + "           <Tag>"
+      //         + "             <Key>key1</Key>"
+      //         + "             <Value>value1</Value>"
+      //         + "           </Tag>"
+      //         + "           <Tag>"
+      //         + "             <Key>key1</Key>"
+      //         + "             <Value>value1</Value>"
+      //         + "           </Tag>"
+      //         + "       </And>"
+      //         + "    </Filter>"
+      //         + "    <Destination>"
+      //         + "      <Bucket>arn:aws:s3:::exampletargetbucket</Bucket>"
+      //         + "    </Destination>"
+      //         + "  </Rule>"
+      //         + "</ReplicationConfiguration>";
+
+      /* MinIO server specific configuration. */
+      String config =
+          "{"
+              + "  \"ReplicationArn\" :\"arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket\","
```

Comment:
ReplicationArn was renamed to Role, so what you originally had is good - you can remove any references to ReplicationArn

Reply:
No problem. Removed all minio server specific references.